spec.md

```diff
@@ -0,0 +1,251 @@
+
+## Table of Contents
+
+- [Scope](#scope)
+  - [Future](#future-scope)
+- [Defining OCI Artifact Types](#defining-oci-artifact-types)
+- [Defining a Unique Artifact Type](#defining-a-unique-artifact-type)
+- [Defining Supported Layer Types](#defining-supported-layer-types)
+- [Optional: Defining Config Schema](#optional-defining-config-schema)
+- [Optional: Artifact Publisher Manifest](#optional-defining-artifact-publisher-manifests)
+  - [Artifact Publisher Manifest Example](#artifact-publisher-manifest-type-example) as a [Well Known Type][def-well-known-types]
+
+## Scope
+
+[Artifacts v1.0][artifacts] is dependent on [OCI manifest][image-manifest] 1.0 to represent individual artifacts including [OCI Image][image-spec], [Helm][helm] and [Singularity][singularity].
+
+### Future Scope
+
+Future versions of artifacts will support new artifacts that represent collections of artifacts using [OCI Index][image-index].
+
+To support artifacts that represent a collection of other artifacts, a means to identify an index as a type of artifact will be required.
+
+## Defining OCI Artifact Types
+
+As registries become content addressable distribution points, tools that pull artifacts must know if they can operate on the artifact. Artifact types are equivalent to file extensions. When users open files, the host operating system typically launches the appropriate program. When users open a file, from within a program, the open dialog filters to the supported types. When search or security software scan the contents of a storage solution, the software must to know how to process the different types of content. When users view the contents of a storage solution, they see the textual and visual indications of the type. OCI Artifacts provides these core capabilities to [OCI distribution spec][distribution-spec] based registries.
+
+Authoring an OCI Artifacts involves the following steps:
+
+- Define a unique type
+- Define the format for other tools to operate upon the type
+- Define human elements, such as an icon and localized string to be displayed to users
+- Optionally, publish as a [well-known type][def-well-known-types] for registries to consume
+
+Defining a unique type, string and logo; artifacts can be represented as the following:
+
+|Icon|Artifact|`config.mediaType`|
+|-|-|-|
+|<img src="https://github.com/opencontainers/artwork/blob/master/oci/icon/color/oci-icon-color.png?raw=true" width=30x>|[OCI Image][image-spec]|`application/vnd.oci.image.config.v1+json`|
+|<img src="https://github.com/helm/helm-www/blob/master/themes/helm/static/img/apple-touch-icon.png?raw=true" width=30x>|[Helm Chart](https://helm.sh)|`application/vnd.cncf.helm.chart.config.v1+json`|
+|<img src="https://github.com/sylabs/singularity/blob/master/docs/logos/singularity_v3.png?raw=true" width=30x>|[Singularity][singularity], by [Sylabs][sylabs]|`application/vnd.sylabs.sif.config.v1+json`|
+
+## Defining a Unique Artifact Type
```

Comment:
Although this section should be specifying how to define an artifact type (e.g. a picture, an application including a default configuration, ...), it looks as if it would be specifying only the configuration (at least that's how I interpreted it the first time I read it). Although the artifact can be a configuration (e.g. a HelmChart), in the case of an OCI-image the difference between the artifact and the configuration is clearer.

Therefore I'd rather remove any misleading reference to a configuration in this section.

Reply:
Thanks, this is a good point to not introduce too much too early. That said, I couldn't find a way to "paint the picture" without providing some detail.
I've added this to help make the correlation between the file extension and the use of `config.mediaType`
> The manifest `config.mediaType` is the equivalent to a file extension, enabling artifact type differentiation.

Does that help?